scorecard/tests/test_scorecard.py

```diff
@@ -76,65 +76,93 @@ def test_data_explorer_links(self):
         selenium = self.selenium
         selenium.get("%s%s" % (self.live_server_url, "/profiles/municipality-BUF-buffalo-city/"))
         self.wait_until_text_in(".page-heading__title", "Buffalo City")
-        link_class = '.is--pre-2019-20 a'
+        post_class = '.indicator-calculation__formula-actual.is--post-2019-20 a'
+        pre_class = '.indicator-calculation__formula-actual.is--pre-2019-20 a'
 
         # Cash Balance
-        element = selenium.find_elements_by_css_selector(link_class)[0].get_attribute("href")
-        self.assertIn('http://portal:8002/table/cflow/?municipalities=BUF&year=2019&items=4200&amountType=AUDA', element)
+        element = selenium.find_elements_by_css_selector('#cash-balance ' + post_class)
+        self.assertIn('http://portal:8002/table/cflow_v2/?municipalities=BUF&year=2019&items=0430&amountType=AUDA', element[0].get_attribute("href"))
+        element = selenium.find_elements_by_css_selector('#cash-balance ' + pre_class)
+        self.assertIn('http://portal:8002/table/cflow/?municipalities=BUF&year=2019&items=4200&amountType=AUDA', element[0].get_attribute("href"))
 
         # Cash Coverage
-        element = selenium.find_elements_by_css_selector(link_class)[1].get_attribute("href")
-        self.assertIn('http://portal:8002/table/cflow/?municipalities=BUF&year=2019&items=4200&amountType=AUDA', element)
-        element = selenium.find_elements_by_css_selector(link_class)[2].get_attribute("href")
-        self.assertIn('http://portal:8002/table/incexp/?municipalities=BUF&year=2019&items=4600&amountType=ADJB', element)
+        element = selenium.find_elements_by_css_selector('#cash-coverage ' + post_class)
+        self.assertIn('http://portal:8002/table/cflow_v2/?municipalities=BUF&year=2019&items=0430&amountType=AUDA', element[0].get_attribute("href"))
```

Comment:
Is there a way to set the base host for testing rather than specifying it in every `assertIn`? With Cypress you use:
```
module.exports = defineConfig({
  e2e: {
    baseUrl: 'http://localhost:3000'
  }
})
```

Reply:
I couldn't find a way to do this with Selenium, for a start I moved the data portal URL to a variable